modules/kafka/src/main/java/com/devonfw/module/kafka/common/messaging/trace/impl/MessageTraceHeaders.java

```diff
@@ -0,0 +1,21 @@
+package com.devonfw.module.kafka.common.messaging.trace.impl;
+
+/**
+ * @author ravicm
+ *
+ */
+public abstract class MessageTraceHeaders {
```

Comment:
Devon does not use b3 headers currently but a simple correlation ID (see com.devonfw.module.logging.common.api.LoggingConstants). Please align the implementation with that.